app/views/wiki/_wikis.html.erb

```diff
@@ -21,6 +21,7 @@
         <td><%= distance_of_time_in_words(Time.at(wiki.latest.created_at), Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %> <%= raw t('wiki._wikis.by') %>
         <a href="/profile/<%= wiki.latest.author.name %>"><%= wiki.latest.author.name %></a></td>
         <td><%= wiki.revisions.length %></td>
+          # Removing The Class d-none d-md-inline
```

Comment:
Hi! These inline comments are really helpful, but unfortunately they'll appear on the page -- an HTML comment is in format: `<!-- here is the comment -->` -- if you can switch them all to this syntax, we should be good! Thanks for the great fix!